### DISCUSSION TITLE
Ability to define how legend should be sorted

### DISCUSSION THREAD

**Discussion by gtirloni** [grafana/grafana#73327] [Dashboards, Panels & Visualizations]
In Grafana 10, the time series panel does not allow one to define how the legend should be sorted:

![image](https://github.com/grafana/grafana/assets/5902830/b1c2ab61-fdbb-487b-8892-7000c6a9d848)

It's possible to sort the legend after it's rendered, by clicking on the arrow for the specific column. This is not ideal.

Something like the sorting for the Tooltip should be implemented for the Legend.

![image](https://github.com/grafana/grafana/assets/5902830/337cc9f1-cf81-45b2-9860-c20fcde42cc2)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82569. Feel free to continue the discussion around this there. Thank you!
